Script Analysis Examples/Specification Analysis/SpecAnalysis_ComplainceCalculation.ipynb

```diff
@@ -0,0 +1,316 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Simple Specification Analysis Example\n",
+    "\n",
+    "This example demonstrates a basic compliance calculations for various specs inside products. It uses the **Spec Service** to query, analyze and update the specs with the latest properties."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Imports\n",
+    "\n",
+    "Import Python Modules for executing the notebook. The requests library is used for communicating with various SystemLink Enterprise's endpoints. Scrapbook is used for running notebooks and recording data for the SystemLink Notebook Execution Service.\n",
```

Comment:
We are not using request library anymore, right?

Reply:
removed the import